controllers/details.js

```diff
@@ -1,13 +1,16 @@
 'use strict';
 
 var api = require('../api');
+var helpers = require('../lib/helpers');
 
 module.exports = function renderDetails(req, res) {
   if (req.params && req.params.sysnum) {
-    var username = req.headers['X-Auth-Username'] || req.headers['x-auth-username'];
+    const username = req.headers['X-Auth-Username'] || req.headers['x-auth-username'];
+    const canRedirectToResults = req.query && req.query.multipleResults !== undefined;
```

Comment:
Playing around in node's repl this line will give true, undefined or false.
I think we probably want:
```js
const canRedirectToResults = (req.query && req.query.multipleResults) !== undefined;
```

Which will give a boolean.

What you have at the minute will still work though. (As undefined is still falsy) It's just a bit odd.

Do you want to change it? (I'm more than happy to merge it as it is.)